scripts/aave_borrow.py

```diff
@@ -43,7 +43,7 @@ def main():
     borrow_tx.wait(1)
     print("We borrowed some DAI!")
     get_borrowable_data(lending_pool, account)
-    repay_all(AMOUNT, lending_pool, account)
+    repay_all(amount_dai_to_borrow, lending_pool, account)
```

Comment:
```suggestion
    repay_all(amount_dai_to_borrow * Decimal('1.01'), lending_pool, account)
```
I've tried your solution and for me I had to make one more change to repay debt completely.
In [documentation](https://docs.aave.com/developers/the-core-protocol/lendingpool#repay) aave recommends sending back slightly more: "In case of repayments on behalf of another user, it's recommended to send an _amount slightly higher than the current borrowed amount."

Reply:
Yes, i agree that it will be best to send slightly more dai because the acumulated interest, but if you borrow 10 dai for example, you can't return 10.1 dai. You are assuming that in the wallet you have some more extra dai.
If you tray to repay 10.1 and you only have 10 the transaction will revert (i think, i didn't test it)

Reply:
You’re right, this solution requires DAI in wallet. So how I see possible options:
1. Rename function to “repay_max”, because you can’t repay all without DAI this way.
2. Add comments about required DAI in wallet.

Reply:
Sorry for the delay, i added some comments on the repay_all function, you are right. But i didn't change the name of the function because if this changes are included on the main branch, maybe the people how sees the Youtube tutorial will be a little lost if they see a function with a different name and it behaves different too.